znai-java/src/main/java/com/twosigma/znai/java/extensions/JavaIncludePlugin.java

```diff
@@ -76,7 +76,7 @@ public JavaIncludeResult process(JavaCode javaCode) {
     }
 
     private String extractContent(JavaCode javaCode) {
-        if (entry == null && entries == null) {
+        if (entry == null && entries.isEmpty()) {
```

Comment:
Redundant.
JavaPluginIncludeBase.process() already checks the condition.

Reply:
This condition is slightly different. It says if no entry or entries is defined use the entire java file to render and not a piece of a method. It was buggy for the last 2 months and was rendering nothing as this `entires` was changed to be an empty array instead of null. There was no test for this case, hence I added it now.

Reply:
Got you.